From ad4fe85f1cfcad5c1f299a0741838ce91c87c4b9 Mon Sep 17 00:00:00 2001
From: Ming Tan <ming.tan@intel.com>
Date: Mon, 19 Mar 2018 13:11:14 +0800
Subject: [PATCH 4/4] Let the kernelflinger assume the BIOS already enable
 secure boot.

It is useful when the BIOS does not support secure boot, but still
want kernelflinger maybe set the verify boot state to green.

JIRA: AIA-418
Test: Verify boot state set to green.

Signed-off-by: Ming Tan <ming.tan@intel.com>
---
 libkernelflinger/security_efi.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/libkernelflinger/security_efi.c b/libkernelflinger/security_efi.c
index a1be3f5..e245a4b 100755
--- a/libkernelflinger/security_efi.c
+++ b/libkernelflinger/security_efi.c
@@ -83,10 +83,10 @@ EFI_STATUS set_platform_secure_boot(__attribute__((unused)) IN UINT8 secure)
  */
 BOOLEAN is_platform_secure_boot_enabled(VOID)
 {
-	EFI_GUID global_guid = EFI_GLOBAL_VARIABLE;
+	return TRUE;
+/*	EFI_GUID global_guid = EFI_GLOBAL_VARIABLE;
 	EFI_STATUS ret;
 	UINT8 value;
-
 	ret = get_efi_variable_byte(&global_guid, SETUP_MODE_VAR, &value);
 	if (EFI_ERROR(ret))
 		return FALSE;
@@ -98,7 +98,7 @@ BOOLEAN is_platform_secure_boot_enabled(VOID)
 	if (EFI_ERROR(ret))
 		return FALSE;
 
-	return value == 1;
+	return value == 1;*/
 }
 
 BOOLEAN is_eom_and_secureboot_enabled(VOID)
-- 
1.9.1

